server/test/integration/vitalsTest/handleHeartbeatTest.js

```diff
@@ -0,0 +1,176 @@
+// Third-party dependencies
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const sinonChai = require('sinon-chai')
+const { afterEach, beforeEach, describe, it } = require('mocha')
+const sinon = require('sinon')
+
+// In-house dependencies
+const { helpers } = require('brave-alert-lib')
+const { braveAlerter, db, redis, server } = require('../../../index')
+const { clientFactory, locationFactory } = require('../../../testingHelpers')
+
+chai.use(chaiHttp)
+chai.use(sinonChai)
+
+const expect = chai.expect
+const sandbox = sinon.createSandbox()
+const testLocation1Id = 'TestLocation1'
+const radar_coreID = 'radar_particlecoreid1'
+const firstLowBatteryAlert = '2021-03-09T19:37:28.176Z'
+
+async function normalHeartbeat(coreId) {
+  try {
+    await chai.request(server).post('/api/heartbeat').send({
+      coreid: coreId,
+      data: `{"doorMissedMsg": 0, "doorLowBatt": false, "doorLastHeartbeat": 1000, "resetReason": "NONE", "states":[]}`,
+    })
+    await helpers.sleep(50)
+  } catch (e) {
+    helpers.log(e)
+  }
+}
+
+async function lowBatteryHeartbeat(coreId) {
+  try {
+    await chai.request(server).post('/api/heartbeat').send({
+      coreid: coreId,
+      data: `{"doorMissedMsg": 0, "doorLowBatt": true, "doorLastHeartbeat": 1000, "resetReason": "NONE", "states":[]}`,
+    })
+    await helpers.sleep(50)
+  } catch (e) {
+    helpers.log(e)
+  }
+}
+
+describe('siren.js integration tests: handleSirenAddressed', () => {
```

Comment:
I think this is a forsaken remnant of a copy paste

Reply:
Thanks! I'll fix this when I update the migration ID after Dave's pull request is merged.

Reply:
Done.